readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -3,51 +3,43 @@
 var constants = require('./constants');
 var rtddata = require('./rtd-data');
 
-var bowser = require('bowser');
-
 var rtd;
 
-var EXPLICIT_PLACEMENT_SELECTOR = '#ethical-ad-placement';
+var EXPLICIT_PLACEMENT_SELECTOR = "[data-ea-publisher]";
+
+// Old way to control the exact placement of an ad
+var OLD_EXPLICIT_PLACEMENT_SELECTOR = '#ethical-ad-placement';
 
 
 /*
- * Create an explicit placement if the project has specified one
+ * Inject the EthicalAds ad client
  */
-function create_explicit_placement() {
-    var element_id = 'rtd-explicit';
-    var display_type = constants.PROMO_TYPES.LEFTNAV;
-    var class_name;         // Used for theme specific CSS customizations
-
-    if(rtd.is_rtd_like_theme()) {
-        class_name = 'ethical-rtd ethical-dark-theme';
-    } else {
-        class_name = 'ethical-alabaster';
-    }
-
-    if ($(EXPLICIT_PLACEMENT_SELECTOR).length > 0) {
-        $('<div />').attr('id', element_id)
-            .addClass(class_name).appendTo(EXPLICIT_PLACEMENT_SELECTOR);
-
-        return {
-            'div_id': element_id,
-            'display_type': display_type,
-        };
-    }
-    return null;
+function inject_ads_client() {
+    var script = document.createElement("script");
+    script.src = "https://media.ethicalads.io/media/client/beta/ethicalads.min.js";
```

Comment:
We should rebase this to get the latest updates.

Reply:
You mean regenerate this beta client?

Reply:
Yea, so it has the latest other client features.

Reply:
I built the latest beta client.